tests/prefect/test_tasks_sqlite.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pandas
+from viadot.tasks.sqlite_tasks import RunSQL, LoadDF
+from collections import defaultdict
+import pandas as pd
+import os
+
+TABLE = "test"
+DB_PATH = ':memory:'
```

Comment:
what is this for? It seems you are not using this variable in the `LoadDF()` Task class...